packages/tab/TabRipple.tsx

```diff
@@ -20,47 +20,41 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import classnames from 'classnames';
-import {withRipple} from '@material/react-ripple';
+import * as Ripple from '@material/react-ripple';
```

Comment:
By convention, I usually leave a newline between packages I didn't author and ones I did.

```typescript
import * as React from 'react';
import * as classnames from 'classnames';

import {withRipple} from '@material/react-ripple';
import {somethingElse} from '@material/react-something-else';

import {something} from './a_local_file';
import {anotherThing} from './another_local_file';
```